bin/calcStats

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Getopt::Long;
+use Bio::Seq;
+use Bio::SeqIO;
+use Bio::Tools::SeqStats;
+use Bio::Tools::pICalculator;
+
+use constant DEBUG => 0;
+
+my ($dataset, $outFile);
+&GetOptions(
+    "dataset=s"       => \$dataset,
+    "outFile=s"       => \$outFile);
+
+unless ($dataset && $outFile ) { die "args dataset and outfile are required\n" };
+unless (-e $dataset) { die "$dataset not found\n" };
+
+open (OUT, ">>$outFile") or die "Can't open '$outFile' for writing\n";
+
+##########################
+##### "main()" ###########
+
+if ( -f $dataset ) {
+    warn "Processing $dataset\n" if DEBUG;
+    processOneFile($dataset, \*OUT);
+} elsif ( -d $dataset ) {
```

Comment:
i think the input for this script is one file.  why the directory processing?

Reply:
copied from orfFinder

Reply:
i just updated orfFinder to only take an "inputFile".  There is no need to have the directory processing there.  i had just moved over the existing script but better to remove it